zkevm-circuits/src/evm_circuit/execution/error_oog_log.rs

```diff
@@ -255,49 +290,48 @@ mod test {
             .write_op(terminator)
         };
 
-        Account::mock_100_ether(bytecode)
+        Account {
+            address: Address::repeat_byte(0xfe),
+            balance: Word::from(10).pow(20.into()),
+            code: bytecode.code().into(),
+            ..Default::default()
+        }
```

Comment:
```suggestion
        Account::mock_100_ether(bytecode)
```

Reply:
updated in [2236916](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1564/commits/22369163147c2f20c622f8521a59b1ef24e13352) thanks !